src/sentry/static/sentry/app/views/eventsV2/table/queryField.tsx

```diff
@@ -459,14 +460,10 @@ class BufferedInput extends React.Component<InputProps, InputState> {
 }
 
 // Set a min-width to allow shrinkage in grid.
-const StyledInput = styled('input')`
+const StyledInput = styled(Input)`
   /* Match the height of the select boxes */
   height: 37px;
   min-width: 50px;
-
-  &:not([disabled='true']):invalid {
-    border-color: ${p => p.theme.red};
-  }
```

Comment:
Actually don't know if it gets this or not, but I dunno if we do this kind of thing anywhere else

Reply:
This is from an earlier iteration that used HTML5 validation errors and input type=number. That approach didn't work out as number inputs have clunky UX.